airflow/providers/apache/hive/CHANGELOG.rst

```diff
@@ -19,6 +19,15 @@
 Changelog
 ---------
 
+1.0.2
+.....
+
+Bug fixes
+~~~~~~~~~
+
+* ``Corrections in docs and tools after releasing provider RCs (#14082)``
```

Comment:
Is it actually worth releasing this if it's the only change?

Reply:
I believe so. Since it costs us nothing but it improves the documentation which is released as part of the providers - yes I thought about it yesterday and while I initially excluded them, I think it is definitely worth to release it. It is important to get the updated user documentation in the hands of the users I believe. 

It has zero risk, practically no consequences for our users (they will just install newer version when they do). It costs us almost nothing for the batch release. 

Yes. I truly believe it is worth to release them.